loc/de_de.json

```
@@ -295,37 +206,44 @@
     "input_done": "Fertig",
     "input_paste": "Einfügen",
     "input_total": "Gesamt:",
-    "no_tx_signing_in_progress": "Keine Transaktionsignierung in Arbeit",
-    "open_settings": "Einstellungen Öffnen",
     "permission_camera_message": "Wir brauchen Deine Erlaubnis um die Kamera zu nutzen.",
     "permission_camera_title": "Erlaubnis die Kamera zu benutzen",
-    "permission_storage_denied_message": "BlueWallet kann die Datei nicht speichern. Bitte öffne die Systemeinstellungen und erteile der App BlueWallt das Recht den internen Speicher zu verwenden.",
+    "open_settings": "Einstellungen Öffnen",
```

This is wrong, it should be this:
```suggestion
    "open_settings": "Einstellungen öffnen",
```
In German only nouns are capitalized, verbs are not.